examples/tutorials/Part 11b - Secure Classification with Syft Keras and TFE: Secure Model Serving.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<p>- extra comma "...define your Syft Keras model, like we did..."</p><p><br></p><p>- "share" doesn't tell us what is happening in the description (aside from plugging TFE). It should be talking about protocols (SPDZ, SecureNN, etc.), not libraries.</p><p><br></p><p>- "serve" also doesn't say what's happening. It says "what you'll be ready for" but it needs to say "this opens up an endpoint on X ports/hostnames... etc."</p><p><br></p><p>- "shutdown_workers" - this is the first time we mention a "worker". This should be mentioned earlier.</p><p><br></p><p>Some of these can be addressed by moving the bottom paragraph up to the top and then referencing it.</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>